idx/cassandra/cassandra.go

```diff
@@ -196,9 +201,11 @@ func (c *CasIdx) Init() error {
 		return err
 	}
 
-	for i := 0; i < numConns; i++ {
-		c.wg.Add(1)
-		go c.processWriteQueue()
+	if updateCassIdx {
+		for i := 0; i < numConns; i++ {
+			c.wg.Add(1)
```

Comment:
we can move this to before the loop and make it a single call: `c.wg.Add(numConns)`